pennylane/qchem/convert.py

```diff
@@ -1075,18 +1115,9 @@ def _dmrg_state(wavefunction, tol=1e-15):
         row.append(stra)
         col.append(strb)
 
-        # compute and fix parity to stick to pyscf notation
-        lsta = np.array(list(map(int, bin(stra)[2:])))[::-1]
-        lstb = np.array(list(map(int, bin(strb)[2:])))[::-1]
-
-        # pad the relevant list
-        maxlen = max([len(lsta), len(lstb)])
-        lsta = np.pad(lsta, (0, maxlen - len(lsta)))
-        lstb = np.pad(lstb, (0, maxlen - len(lstb)))
-
-        which_occ = np.where(lsta == 1)[0]
-        parity = (-1) ** np.sum([np.sum(lstb[: int(ind)]) for ind in which_occ])
-        dat.append(parity * coeffs[ii])
+        # slater determinant sign convention of block2 (physicist,
+        # interleave spin-up/down operators) is consistent with pennylane
+        dat.append(coeffs[ii])
```

Comment:
Isn't `dat` just duplicating (or maybe truncating) `coeffs`? Can't we directly use `coeffs[:ii + 1]` instead of `dat` for building `dict_fcimatr`?